compose/ui/ui-text/src/nativeMain/kotlin/androidx/compose/ui/text/intl/NativePlatformLocale.native.kt

```diff
@@ -35,7 +35,10 @@ internal class NativeLocale(val locale: NSLocale) : PlatformLocale {
 internal actual fun createPlatformLocaleDelegate(): PlatformLocaleDelegate =
     object : PlatformLocaleDelegate {
         override val current: LocaleList
-            get() = LocaleList(listOf(Locale(NativeLocale(NSLocale.currentLocale))))
+            get() {
+                val locale = NSLocale(NSLocale.preferredLanguages.first() as String)
```

Comment:
🤔 
If it's a list and we need to return the list, should we pick up only first element?